teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md

```diff
@@ -56,6 +56,43 @@ EnableAcsUsers       : False
 
 ## PARAMETERS
 
+### -Filter
```

Comment:
Parameters have been added to the parameters list but not to the syntax block. Please update the syntax block.

Reply:
Addressed